src/components/banner/MainBanner.tsx

```
@@ -2,130 +2,38 @@
 
 import Image from 'next/image';
 import { useRouter } from 'next/navigation';
-import React, { useState, useEffect, useRef, useCallback } from 'react';
+import React from 'react';
 
-import BANNER_DATA, { BannerItem } from './bannerData';
+import BANNER_DATA from './bannerData';
 import * as styles from './mainBanner.css';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)